template/setup/index.js

```diff
@@ -18,25 +36,39 @@ async function setup() {
   }
 
   console.log('\n📝  Configuring project display name and bundle identifier...');
-  const { displayName, bundleIdentifer } = await prompt([
-    {
-      type: 'input',
-      name: 'displayName',
-      message: 'App Display Name (ProjectName):',
-    },
-    {
-      type: 'input',
-      name: 'bundleIdentifer',
-      message: `App Bundle Identifier (com.${displayName.toLowerCase()}.mobile):`,
-    },
-  ]);
+  const { displayName } = await prompt({
+    type: "input",
+    name: "displayName",
+    message: "App Display Name (ProjectName):",
+    initial: "HelloWorld"
+  });
+
+  const { bundleIdentifer } = await prompt({
+    type: 'input',
+    name: 'bundleIdentifer',
+    message: 'App Bundle Identifier',
+    initial: displayName.toLowerCase() || 'helloworld',
+  });
 
   const { confirmed } = await prompt({
     type: 'confirm',
     name: 'confirmed',
     message: `Continue with => App Display Name: ${displayName} | App Bundle Identifer: ${bundleIdentifer}`,
   });
 
+  if (!confirmed) {
+    console.log(
+      '\n Cannot continue without choosing a display name and app bundle identifier...',
+    );
+    console.log('\n Exiting setup...');
```

Comment:
we added a `return` here when pairing right?

Reply:
should be back in now!